### PR TITLE
[DOC-3744] SMP 0.8.3, 798xx patch release notes

### PR DESCRIPTION
**Pull request by brian-f-harness**
SMP 0.8.3, 798xx patch release notes

For reviewers, a preview is available.

https://650b663515bafa59583fc6cb--harness-developer.netlify.app/release-notes/self-managed-enterprise-edition#patches

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/self-managed-enterprise-edition.md [hunk 1]
```diff
@@ -564,6 +564,106 @@ You are missing the following permission: "View default settings" in Account sco
...
 
+##### 0.8.3
+
+This release includes the following Harness module and component versions.
+
+| **Name** | **Version** |
+| :-- | :--: |
+| Helm Chart | [0.8.3](https://github.com/harness/helm-charts/releases/tag/harness-0.8.3) |
+| Air Gap Bundle | [0.8.3](https://console.cloud.google.com/storage/browser/smp-airgap-bundles/harness-0.8.3) |
+| NG Manager | 79821 |
```
Comment (swapnil-harness) [inline release-notes/self-managed-enterprise-edition.md:575]: This should be ng-manager-signed:79823

File: release-notes/self-managed-enterprise-edition.md [hunk 2]
```diff
@@ -564,6 +564,106 @@ You are missing the following permission: "View default settings" in Account sco
...
+| CI Manager | 4903 |
```
Comment (swapnil-harness) [inline release-notes/self-managed-enterprise-edition.md:576]: This should be ci-manager-signed:4905

File: release-notes/self-managed-enterprise-edition.md [hunk 3]
```diff
@@ -564,6 +564,106 @@ You are missing the following permission: "View default settings" in Account sco
...
+| Pipeline Service | 1.37.13 |
+| Platform Service | 79601 |
+| Access Control Service | 79400 |
+| Change Data Capture | 79819 |
+| Test Intelligence Service | release-177 |
+| NG UI | 0.353.10 |
```
Comment (swapnil-harness) [inline release-notes/self-managed-enterprise-edition.md:582]: This should be nextgenui-signed:0.353.17

**Review by swapnil-harness — DISMISSED** [review]
Added few comments, everything else looks good

File: release-notes/self-managed-enterprise-edition.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 title: Self-Managed Enterprise Edition release notes
 sidebar_label: Self-Managed Enterprise Edition
 tags: [NextGen, "self-managed-ee"]
-date: 2023-08-23T10:00
+date: 2023-09-19T10:00
```
Comment (brian-f-harness) [inline release-notes/self-managed-enterprise-edition.md:5]:
```suggestion
date: 2023-09-20T10:00
```

File: release-notes/self-managed-enterprise-edition.md [hunk 5]
```diff
@@ -564,6 +564,106 @@ You are missing the following permission: "View default settings" in Account sco
...
+   monitoring.<region>.amazonaws.com
+   ```
+
+#### Chaos Engineering
+
+- Added support for Chaos Engineering version 1.19.x. (CHAOS-2544)
+
+#### Continuous Delivery
+
+- Added the GitOps feature flag at the org level. (CDS-79545)
```
Comment (pratmit) [inline release-notes/self-managed-enterprise-edition.md:645]: Added the functionality to support Gitops section at Org Level

File: release-notes/self-managed-enterprise-edition.md [hunk 6]
```diff
@@ -564,6 +564,106 @@ You are missing the following permission: "View default settings" in Account sco
...
+   ec2.<region>.amazonaws.com
+
+   ECS
+   ecs.<region>.amazonaws.com
+
+   cloudwatch
+   monitoring.<region>.amazonaws.com
+   ```
+
+#### Chaos Engineering
```
[the remark this answers is not in the document]
Comment (brian-f-harness) [reply release-notes/self-managed-enterprise-edition.md:639]: added

File: release-notes/self-managed-enterprise-edition.md [hunk 7]
```diff
@@ -564,6 +564,106 @@ You are missing the following permission: "View default settings" in Account sco
...
+  "gs://smp-airgap-bundles/harness-0.8.3/ci_images.tgz" \
+  "gs://smp-airgap-bundles/harness-0.8.3/ff_images.tgz" \
+  "gs://smp-airgap-bundles/harness-0.8.3/platform_images.tgz" \
+  "gs://smp-airgap-bundles/harness-0.8.3/sto_images.tgz" \
+  .
+```
+
+#### New features and enhancements
+
+#### Self-Managed Enterprise Edition
```
Comment (michaelcretzman) [inline release-notes/self-managed-enterprise-edition.md:604]: these heading levels are the same. Is Self-Managed EE supposed to be subordinate to New Features and enhancements?

File: release-notes/self-managed-enterprise-edition.md [hunk 8]
```diff
@@ -564,6 +564,106 @@ You are missing the following permission: "View default settings" in Account sco
...
+
+- Updated CI Manager and the delegate to improve performance testing.(SMP-2022)
```
Comment (michaelcretzman) [inline release-notes/self-managed-enterprise-edition.md:606]:
```suggestion
- Updated CI Manager and the delegate to improve performance testing. (SMP-2022)
```

File: release-notes/self-managed-enterprise-edition.md [hunk 9]
```diff
@@ -564,6 +564,106 @@ You are missing the following permission: "View default settings" in Account sco
...
+
+- Added support for Chaos Engineering version 1.19.x. For more information, go to the [Chaos Engineering release notes](/release-notes/chaos-engineering/#latest-version-1192). (CHAOS-2544)
+
+#### Continuous Delivery
+
+- Added functionality to support GitOps section at Org level. (CDS-79545)
+
+#### Continuous Integration
+
+- Added support to set the `LOG_STREAMING _URL` from Harness Delegate by default for CI pipelines. If the delegate is outside of the cluster, the fallback log service URL is `LOG_SERVICE_ENDPOINT` from CI. (CI-9262, ZD-48974)
```
Comment (michaelcretzman) [inline release-notes/self-managed-enterprise-edition.md:649]:
```suggestion
- To support  CI pipelines, Harness added support to set the Harness Delegate environment variable `LOG_STREAMING _URL` by default. If the delegate is outside of the cluster, the fallback log service URL is `LOG_SERVICE_ENDPOINT` from CI. (CI-9262, ZD-48974)
```

**Review by michaelcretzman — APPROVED** [review]
Approved with minor suggestions.

File: release-notes/self-managed-enterprise-edition.md (hunk 7 above)
Comment (brian-f-harness) [reply release-notes/self-managed-enterprise-edition.md:604]: Nice catch. Thanks, @michaelcretzman

Comment (brian-f-harness) [inline release-notes/self-managed-enterprise-edition.md:604]:
```suggestion
##### Self-Managed Enterprise Edition
```

File: release-notes/self-managed-enterprise-edition.md [hunk 10]
```diff
@@ -564,6 +564,106 @@ You are missing the following permission: "View default settings" in Account sco
...
+- Updated CI Manager and the delegate to improve performance testing.(SMP-2022)
+
+- Added the following rule to the `log-service` virtual service for performance testing. (SMP-2033)
+
+   ```yaml
+   - uri:
+        prefix: /log-service//
+   ```
+
+#### Cloud Cost Management
```
Comment (brian-f-harness) [inline release-notes/self-managed-enterprise-edition.md:615]:
```suggestion
##### Cloud Cost Management
```

File: release-notes/self-managed-enterprise-edition.md [hunk 11]
```diff
@@ -564,6 +564,106 @@ You are missing the following permission: "View default settings" in Account sco
...
+   ec2.<region>.amazonaws.com
+
+   ECS
+   ecs.<region>.amazonaws.com
+
+   cloudwatch
+   monitoring.<region>.amazonaws.com
+   ```
+
+#### Chaos Engineering
```
Comment (brian-f-harness) [inline release-notes/self-managed-enterprise-edition.md:639]:
```suggestion
##### Chaos Engineering
```

File: release-notes/self-managed-enterprise-edition.md [hunk 12]
```diff
@@ -564,6 +564,106 @@ You are missing the following permission: "View default settings" in Account sco
...
+
+- Added support for Chaos Engineering version 1.19.x. For more information, go to the [Chaos Engineering release notes](/release-notes/chaos-engineering/#latest-version-1192). (CHAOS-2544)
+
+#### Continuous Delivery
```
Comment (brian-f-harness) [inline release-notes/self-managed-enterprise-edition.md:643]:
```suggestion
##### Continuous Delivery
```

File: release-notes/self-managed-enterprise-edition.md [hunk 13]
```diff
@@ -564,6 +564,106 @@ You are missing the following permission: "View default settings" in Account sco
...
+
+- Added functionality to support GitOps section at Org level. (CDS-79545)
+
+#### Continuous Integration
```
Comment (brian-f-harness) [inline release-notes/self-managed-enterprise-edition.md:647]:
```suggestion
##### Continuous Integration
```

File: release-notes/self-managed-enterprise-edition.md [hunk 14]
```diff
@@ -564,6 +564,106 @@ You are missing the following permission: "View default settings" in Account sco
...
+
+- Added support to set the `LOG_STREAMING _URL` from Harness Delegate by default for CI pipelines. If the delegate is outside of the cluster, the fallback log service URL is `LOG_SERVICE_ENDPOINT` from CI. (CI-9262, ZD-48974)
+
+#### Custom Dashboards
```
Comment (brian-f-harness) [inline release-notes/self-managed-enterprise-edition.md:651]:
```suggestion
##### Custom Dashboards
```

File: release-notes/self-managed-enterprise-edition.md [hunk 15]
```diff
@@ -564,6 +564,106 @@ You are missing the following permission: "View default settings" in Account sco
...
+
+- Updated to an FIPS compliant algorithm for short term caching key generation. (CDB-1107)
+
+#### Early access features
+
+This release does not include any early access features.
+
+#### Fixed issues
+
+#### Continuous Integration
```
Comment (brian-f-harness) [inline release-notes/self-managed-enterprise-edition.md:661]:
```suggestion
##### Continuous Integration
```

**Review by pratmit — COMMENTED** [review]
Added the below comments

**Comment by brian-f-harness** [comment]
> Added the below comments

@pratmit Added with this commit: https://github.com/harness/developer-hub/pull/3348/commits/5eb73eb4b043173530cd7279c052fb74740c2432

Included in the updated preview: https://650b663515bafa59583fc6cb--harness-developer.netlify.app/release-notes/self-managed-enterprise-edition#patches

File: release-notes/self-managed-enterprise-edition.md [hunk 16]
```diff
@@ -564,6 +564,106 @@ You are missing the following permission: "View default settings" in Account sco
...
+   ```
+
+##### Cloud Cost Management
+
+- Updated the Helm chart to support proxy configuration for AWS SDK clients. (CCM-14374)
+
+- Updated the Helm chart to support AWS regions other than the default `us-east-1` region. (CCM-14415)
+   
+   ```
+   S3
```
Comment (nikunj-badjatya) [inline release-notes/self-managed-enterprise-edition.md:622]: We can remove this block of endpoint urls from here. 621 to 637

**Review by nikunj-badjatya — APPROVED** [review]

**Pull request merged**